native/src/components/TtsContainer.tsx

```diff
@@ -93,25 +89,20 @@ const TtsContainer = ({ children }: TtsContainerProps): ReactElement => {
     setIsPlaying(false)
   }
 
-  const playNext = useCallback(
-    (index = sentenceIndex) => {
-      const nextIndex = index + 1
-      if (nextIndex < sentences.length) {
-        setSentenceIndex(nextIndex)
-        play(nextIndex)
-      } else {
-        stop()
-      }
-    },
-    [play, sentenceIndex, sentences.length],
-  )
-
-  const playPrevious = (index = sentenceIndex) => {
-    const previousIndex = index - 1
-    if (previousIndex >= 0) {
-      setSentenceIndex(previousIndex)
-      play(previousIndex)
+  const playNext = useCallback(() => {
+    const nextIndex = sentenceIndex + 1
+    if (nextIndex < sentences.length) {
+      setSentenceIndex(nextIndex)
+      play(nextIndex)
+    } else {
+      stop()
     }
+  }, [play, sentenceIndex, sentences.length])
+
+  const playPrevious = () => {
+    const previousIndex = Math.max(0, sentenceIndex - 1)
```

Comment:
This aligns the behavior to that of most media players to just start the first sentence again

Reply:
I think that what it was before 🫣